modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -262,12 +270,14 @@ namespace {{packageName}}.Client
         /// <param name="name">Parameter name.</param>
         /// <param name="stream">Input stream.</param>
         /// <returns>FileParameter.</returns>
-        public FileParameter ParameterToFile(string name, Stream stream)
-        {
-            if (stream is FileStream)
+        public FileParameter ParameterToFile(string name, Stream stream) {
+            log.Trace("start");
+            if (stream is FileStream) {
+                log.Debug("FileStream");
```

Comment:
Again, you're making assumptions about the logging implementation.

To put my comment more into perspective, logging implementations (log4net, for example) grab the executing method name by getting a reference to the execution stack, going back one entry, and getting the method name. I haven't looked at your recent changes, but with the `Debug` method deferring to a `DebugPartial` someone using a common implementation of getting a method name would end up getting "Debug". So the message would be something like:

```
Debug: FileStream
```

That's easy enough to fix, obviously.

But, if I just wanted to implement `DebugPartial` as:

```
Console.WriteLine(message);
```

There's no timestamp here. If the message was `ParameterToFile: FileStream`, I would think that'd be less of an issue. But with just `FileStream`, it's not telling the logger or the human reading the logs anything about the log message and this makes the message less useful.